buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -359,32 +368,41 @@ class VagrantTestPlugin implements Plugin<Project> {
             }
             vagrantSmokeTest.dependsOn(smoke)
 
-            Task packaging = project.tasks.create("vagrant${boxTask}#packagingTest", BatsOverVagrantTask) {
+            Task batsPackagingTest = project.tasks.create("vagrant${boxTask}#batsPackagingTest", BatsOverVagrantTask) {
                 remoteCommand BATS_TEST_COMMAND
                 boxName box
                 environmentVars vagrantEnvVars
-                dependsOn up, setupBats
+                dependsOn up, setupPackagingTest
                 finalizedBy halt
             }
 
-            TaskExecutionAdapter packagingReproListener = new TaskExecutionAdapter() {
-                @Override
-                void afterExecute(Task task, TaskState state) {
-                    final String gradlew = Os.isFamily(Os.FAMILY_WINDOWS) ? "gradlew" : "./gradlew"
-                    if (state.failure != null) {
-                        println "REPRODUCE WITH: ${gradlew} ${packaging.path} " +
-                            "-Dtests.seed=${project.testSeed} "
-                    }
-                }
+            TaskExecutionAdapter batsPackagingReproListener = createReproListener(project)
+            batsPackagingTest.doFirst {
+                project.gradle.addListener(batsPackagingReproListener)
             }
-            packaging.doFirst {
-                project.gradle.addListener(packagingReproListener)
+            batsPackagingTest.doLast {
+                project.gradle.removeListener(batsPackagingReproListener)
             }
-            packaging.doLast {
-                project.gradle.removeListener(packagingReproListener)
+            if (project.extensions.esvagrant.boxes.contains(box)) {
+                packagingTest.dependsOn(batsPackagingTest)
+            }
+
+            Task groovyPackagingTest = project.tasks.create("vagrant${boxTask}#groovyPackagingTest")
+            groovyPackagingTest.dependsOn(up)
+            groovyPackagingTest.finalizedBy(halt)
+            groovyPackagingTest.doLast {
+                // no op
```

Comment:
Why is this needed?

Reply:
I added the empty groovyPackagingTest task to introduce changes incrementally (the next PR will make this into a `VagrantCommandTask` that executes the test jar)

The empty `doLast` with the comment is just to make it explicit that the task doesn't currently do anything